### PR TITLE
Readd Properties to default AbstractIdentifiable

### PR DESCRIPTION
**Pull request by pl-buiquang**
Signed-off-by: Paul Bui-Quang <paul.buiquang@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** 
https://github.com/powsybl/powsybl-core/issues/1039

### CODE REVIEW

File: iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java [hunk 1]
```diff
@@ -60,17 +64,19 @@ public boolean hasProperty(String key) {
 
     @Override
     public String getProperty(String key) {
-        return properties.get(key);
+        Object val = properties.get(key);
+        return val != null ? val.toString() : null;
```
Comment (mathbagu) [inline iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java]: Could the property key really be null? In getPropertyNames() it seems you assume that it’s never null.

Comment (pl-buiquang) [reply iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java]: no the property key is indeed assumed to be null.
Though the value associated to the key may be null.

Comment (geofjamg) [reply iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java]: I think @mathbagu just want you to add a null check on the key as it is not done inside properties.get().

Comment (mathbagu) [reply iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java]: I think the properties class throw the exception (NullPointerException) so it should be ok like this.

**Pull request merged**